node/full_test.go

```diff
@@ -133,6 +133,9 @@ func TestFull_RPCServer(t *testing.T) {
 }
 
 func TestFull_WithRemoteCore(t *testing.T) {
+	// TODO(@Wondertan): Fix core
+	t.Skip("Skip until we fix core")
```

Comment:
Link to issue?

Reply:
https://github.com/celestiaorg/celestia-node/issues/215